tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1141,13 +1151,75 @@ class OpConverterTest : public ::testing::Test {
                                     /*use_calibration=*/false, &logger_,
                                     /*use_implicit_batch=*/trt_mode ==
                                         TrtTestMode::kImplicitBatch,
-                                    /*engine_name=*/"")
+                                    /*engine_name=*/"",
+                                    /*use_explicit_precision=*/false,
+                                    context_.get())
                       .ValueOrDie());
 
     // Reset other related artifacts.
     scope_ = Scope::NewRootScope();
   }
 
+  void CreateContext(const NodeDef& node_def, OpKernel** kernel,
```

Comment:
We are considering the make the converter test file more modular. Would it be possible to place this helper into a separate file, e.g. `variable_test_utils.cc`?

Reply:
This is a class method, as all the smart pointers initialized in it are managed by the test class. Do you suggest making e.g a struct of these pointers in the helper file, owned and passed by pointer by the test method?

Reply:
Thanks for pointing this out. One option would be to
- create a `VariableOpConverterTest` child class with the new member variables and helper functions.
- once we move `OpConverterTest` class definition to a separate file, we can then also place the `VariableOpConverterTest` to its own file. But this shall be a topic of a separate PR.